test/testworker.js

```diff
@@ -231,7 +231,9 @@ export default class TestWorker extends EventEmitter {
     return await Promise.all(graph.tasks.map(async (task) => {
       // Note: that we assume runId 0 here which is fine locally since we know
       // the number of runs but not safe is we wanted to test reruns.
-      return await this.fetchTaskStats(task.taskId, 0);
+      var LocalLiveLog = require('../lib/features/local_live_log');
```

Comment:
Moving it out sounds fine. I made it as a class method, just to centralize where we define the path. I agree, having it in a smaller class sounds better.

I'll change that